docs/team/florentianayuwono.md

```diff
@@ -3,44 +3,48 @@ layout: page
 
-* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=florentianayuwono&breakdown=true)
+**Justification**: This feature improves the user productivity significantly because a user will not need to provide full information upon searching and the app could still return accurate and relevant result based on the query. 
 
-* **Project management**:
-  * Managed releases `v1.3` - `v1.5rc` (3 releases) on GitHub
+**Highlight**: This enhancement replaces the previous `find` functionality. The search feature itself requires careful design on how the algorithm works, especially on the relevancy reduction level.
 
-* **Enhancements to existing features**:
-  * Updated the GUI color scheme (Pull requests [\#33](), [\#34]())
-  * Wrote additional tests for existing features to increase coverage from 88% to 92% (Pull requests [\#36](), [\#38]())
+**Credit**: This feature is adapted from the `find` feature, which only allows full word match result to be returned.
 
-* **Documentation**:
-  * User Guide:
-    * Added documentation for the features `delete` and `find` [\#72]()
-    * Did cosmetic tweaks to existing documentation of features `clear`, `exit`: [\#74]()
-  * Developer Guide:
-    * Added implementation details of the `delete` feature.
+### Code Contribution
 
-* **Community**:
-  * PRs reviewed (with non-trivial review comments): [\#12](), [\#32](), [\#19](), [\#42]()
-  * Contributed to forum discussions (examples: [1](), [2](), [3](), [4]())
-  * Reported bugs and suggestions for other teams in the class (examples: [1](), [2](), [3]())
-  * Some parts of the history feature I added was adopted by several other class mates ([1](), [2]())
+Up until 2 November 2022, I have contributed 1781 LoC (roughly 4,185 additions and 2,481 deletions), changed 36 files and pushed 132 commits. For most recent update, refer to this [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=florentianayuwono&breakdown=true).
 
-* **Tools**:
-  * Integrated a third party library (Natty) to the project ([\#42]())
-  * Integrated a new Github plugin (CircleCI) to the team repo
+### Project Management
 
-* _{you can add/remove categories in the list above}_
+**Releases and Milestones**: 
+* `v1.1`: 1 issue and 1 PR.
+* `v1.2`: 8 issues and 4 PRs.
+* `v1.3`: 18 issues and 6 PRs.
+* `v1.4`: 17 issues and 2 PRs.
+
+**Enhancement to existing features**: 
+* Revamped the GUI theme and CSS structure (Pull requests [\#142](), [\#128]()).
+
+**Documentation**:
+* User Guide:
+  * Added documentation for the `search` feature (Pull requests [\#152](), [\#125](), [\#75]()).
+  * Updated links and overall project documentation (Pull requests [\#152](), [\#125](), [\#82](), [\#12]()).
+* Developer Guide:
+  * Added implementation details and sequence diagram for the `search` feature (Pull requests [\#152](), [\#75]()).
+
+**Community**:
+* 24 PRs reviewed (with 64 review comments).
```

Comment:
Perhaps you could add the link to the filtered list of PRs reviewed by you. (refer to mine)

Reply:
Good suggestion! Thanks :D